src/AttachAddon.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { Terminal, IDisposable } from 'xterm';
+import { IAttachOptions } from '../typings/attach';
```

Comment:
I think we run into issues when importing if we reference typings here, let's just duplicate the interface in this file

Reply:
Fixed.